docs/src/content/v1-migration/find.md

```diff
@@ -0,0 +1,55 @@
+# Typing
+
+When using with a type-predicate, the returned item would now be narrowed to the
+type of the type-predicate, similar to how `filter` with a type-predicate
+narrows the resulting items in the array.
+
```

Comment:
**nit:** let's make this second paragraph to match example order, and for consistency with other `indexed` changes...